packages/dashboard/src/calculator/views/Calculator.vue

```diff
@@ -199,6 +206,7 @@ export default class Calculator extends Vue {
   @Watch("HRU")
   @Watch("balance")
   @Watch("IPV4")
+  @Watch("isCerified")
```

Comment:
I think this should be `isCertified`, no?
```suggestion
  @Watch("isCertified")
```